modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -25,11 +25,17 @@ You can deploy {lvms} using one of the following:
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
 == Requirements
 
-Before you begin deploying {lvms}, ensure that the following requirements are met:
+Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
 
 * You have installed {rh-rhacm-first} on an {product-title} cluster.
 * Every managed cluster has dedicated disks that are used to provision storage.
 
+Before deploying {lvms} on a private CI environment, where virutal machines and storage devices can be reused, ensure that you have wiped the disks that are not in use. 
+[NOTE]
+====
+You can wipe only the disks that are not in use.
+====
```

Comment:
```suggestion
Before deploying {lvms} in a private CI environment where you can reuse virtual machines and storage devices, ensure that you have deleted the disks that are not in use.

[NOTE]
====
You can only delete the disks that are not in use.
====
```

Reply:
@agantony 
Thank you for the review. Just one query on the comment-

When we say "You can delete only the disks that are not in use" - the limitation is on the type of devices that can be deleted.
In case of  "You can only delete the disks that are not in use" - the limitation is the action of deletion.

Or to avoid confusion we can rephrase it to - "You cannot delete the disks that are in use"
Any comments on this approach?

@suleymanakbas91 Could you please confirm if there is a technical difference between "deleting" and "wiping" the disks.

Reply:
"You cannot delete the disks that are in use" --> This SGTM!

Reply:
`wipe` is the common usage, so I'd stick to that.